content-src/static/img/glyph-forward-16.svg

```diff
@@ -7,7 +7,7 @@
       }
     </style>
   </defs>
-  <g id="glyph-forward-16">
+  <g id="glyph-forward-16-white">
```

Comment:
Did we want to give this the same `id` as content-src/static/img/glyph-forward-16-white.svg?